hooks/README.md

```diff
@@ -0,0 +1,98 @@
+# Camio Hooks
+
+## Introduction
+
+Camio provides an image processing pipeline. Videos are uploaded to the Camio cloud from
+a phone or an RTSP camera or a Camio Box. The Camio image pipeline provides a labeling 
+service but also allows you to register your own labeling hooks. A hook is defined as a custom web service that can compute labels from images.
```

Comment:
labeling is one kind of hook, but hooks can also perform other tasks like logging a spreadsheet or sending SMS, etc... so don't limit the hook definition to labeling.